app/models/quote_type.rb

```diff
@@ -0,0 +1,5 @@
+class QuoteType < ApplicationRecord
+  validates :name, length: { maximum: 255 }, presence: true
```

Comment:
Can I have multiple order types with the same name?

Reply:
Not! Order must be unique ?

Reply:
How the users can differentiate the quote types if they have the same name?

Reply:
Right! It need unique.